components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/gen/java/org/wso2/carbon/apimgt/rest/api/core/dto/APIListDTO.java

```diff
@@ -0,0 +1,105 @@
+package org.wso2.carbon.apimgt.rest.api.core.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import org.wso2.carbon.apimgt.rest.api.core.dto.APIInfoDTO;
+import java.util.Objects;
+
+/**
+ * APIListDTO
```

Comment:
This is also an auto generated class when creating a new REST API. So, normally we do not change anything in these classes.